raft/server.cc

```diff
@@ -1167,6 +1168,7 @@ future<read_barrier_reply> server_impl::get_read_idx(server_id leader, seastar::
 }
 
 future<> server_impl::read_barrier(seastar::abort_source* as) {
+    check_not_aborted();
```

Comment:
Same. Or may be put the check into the `wait_for_leader()` instead of separately in `read_barrier`, `add_entry` and `modify_config`.